README.md

```diff
@@ -39,19 +39,18 @@ The following requirements and principles have been used when designing the Elec
 
 ## Trust model
 
-The trust model is currently under development. It is assumed to be a health-specific version of the ICAO Master List concept (see also https://www.who.int/publications/m/item/interim-guidance-for-developing-a-smart-vaccination-certificate) that is both health and COVID-19 specific.
+The trust model is based on the ICAO Master List concept with some minor modifications to improve performance and compatibility. At the center of the model is the [DGC Gateway](https://github.com/eu-digital-green-certificates/dgc-gateway) and the associated [certificate governance](https://github.com/eu-digital-green-certificates/dgc-overview/blob/main/guides/certificate-governance.md). The gateway operates in a B2B model with onboarded countries. Member States are then responsible for publishing their own trust lists publicly on their own national backends.
```

Comment:
All occurrences of "DGC" could be replaced by "DCC", since the GitHub repo links are also renamed (but better check).

Reply:
@dslmeinte GitHub link/repositories were not renamed 

We should stay with https://github.com/eu-digital-green-certificates/dgc-gateway. The [/gcc-gateway](https://github.com/eu-digital-green-certificates/dcc-gateway) gives `404`

Reply:
Oh yeah, not our repos...

OK, then at least the link _text_ should say "DCC" wherever it makes sense.

Reply:
The test should match the name of the repository it's pointing to and unfortunately that still has the old name. I can understand why - we still have issues due to the renaming of the ehn organisation.